components/storage/blockdevice/COMPONENT_FLASHIAP/TESTS/filesystem/fopen/fopen.cpp

```diff
@@ -1247,7 +1257,32 @@ control_t fslittle_fopen_test_00(const size_t call_count)
     (void) call_count;
     int32_t ret = -1;
 
+#if (MBED_CONF_FLASHIAP_BLOCK_DEVICE_SIZE == 0) && (MBED_CONF_FLASHIAP_BLOCK_DEVICE_BASE_ADDRESS == 0xFFFFFFFF)
+
+    size_t flash_size;
+    uint32_t start_address;
+    uint32_t bottom_address;
+    mbed::FlashIAP flash_driver;
+
+    ret = flash_driver.init();
+    TEST_ASSERT_EQUAL(0, ret);
```

Comment:
how does this work? assert equal and the condition below returns to the next test case (does it reach line 1273?).